ci/docker-compose.yml

```diff
@@ -26,3 +27,45 @@ services:
         condition: service_healthy
     volumes:
       - ..:/risingwave
+
+  sink-build-env:
+    image: public.ecr.aws/x5u3w5h6/rw-build-env:v20220729
+    depends_on:
+      mysql:
+        condition: service_healthy
+    volumes:
+      - ..:/risingwave
+    pid: host
+    userns_mode: host
+    network_mode: host
+    # networks:
+    #   nw:
+    #     ipv4_address: 172.24.2.1
+
+  mysql:
+    image: mysql/mysql-server:latest
```

Comment:
would be better if we could specify a version (e.g. `mysql/mysql-server:8.0`)

Reply:
good point, will fix 👍